test/demo/istio-testing/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+# Installs istio-demo profile in istio-testing environment
+# TODO: override as much as possible
```

Comment:
What are we overriding? To get the demo values (resource settings, enabled, etc)?

if so shouldnt we just generate those using helm values?

Reply:
No, we do have the right demo values in test/demo - this is to test how much we can customize the demo with only kustomize.

The 'base' demo is using istio-system - and if we're lucky we may be able to use it as an in-place upgrade from 1.1 ( seems doable - not sure if worth it ).

This is a profile that installs in istio-testing, and will test the limits of kustomization ( == override as much as we can )